### PR TITLE
Reworks Delta Medbay

### PR DESCRIPTION
**Pull request by nmajask**
# Document the changes in your pull request

Changes some parts of the medbay on Delta to be "better" in my opinion, while not completely copying over box medbay and trying new stuff. Most of the new stuff has been inspired by the medbay on Bay as I think it's neat.

The main changes are: 
- that the morgue has been shrunk down 
- surgery has been replaced by the patient rooms
- the psychiatrists/cloning/sec rooms have been moved around and changed
- a few new rooms have been added (treatment center, locker room, and back room)

### View of the entire medbay
image

### Gene/Paramedic Staging Area/Locker Room
Rearranged things so paramedics can enter medbay from their room
image

### Patient Rooms/Security Post/Backroom
The patient rooms replace surgery with each having their own set of tools. Most of the general stuff that would be stored in surgery has been moved to a back room containing many other uncommon items and general maintance stuff for medbay. The psych's office was moved next to the patient treatment rooms as it fit better. The sec checkpoint was moved to above the CMO's office to make room for the treatment center.
image

### Treatment Center
Basically that general area on box medbay that everyone works in. Even has a small reception desk to completely ignore.
image

# Wiki Documentation

Obviously the map images, but other than that no other changes should be needed. 

# Changelog

:cl:  
tweak: reworked Delta medbay
/:cl:

### CODE REVIEW

**Review by patpol4 — APPROVED** [review]
I took a brief look over this and it seems fine.
Not sure if its going to be played with the recent changes though..
